deploy/development/prometheus.yaml

```diff
@@ -0,0 +1,184 @@
+      expr: |
+        sum by (ingress, cluster) (rate(nginx_ingress_controller_requests{exported_namespace="care-arrangement-plan-prod",status=~"5.."}[1m]))
+        /
+        sum by (ingress) (rate(nginx_ingress_controller_requests{exported_namespace="care-arrangement-plan-prod"}[1m]))
+        > 0.05
+      for: 3m
+      labels:
+        severity: care-arrangement-plan-production-alerts
+
+    - alert: SlowResponses
+      annotations:
+        message: For 3 minutes, more than 1% of requests were slower than 5 seconds (during weekday working hours).
+        summary: High P99 latency.
+      expr: |
+        histogram_quantile(
+          0.99,
+          sum by (le, ingress) (
+            rate(
+              nginx_ingress_controller_request_duration_seconds_bucket{exported_namespace="care-arrangement-plan-prod",status!="404",status!="500"}[1m]
+            )
+          )
+        ) > 5
+        AND ON() (7 <= hour() <= 19)
+        AND ON() (1 <= day_of_week() <= 5)
+      for: 3m
+      labels:
+        severity: care-arrangement-plan-production-alerts
+
+    - alert: SlownessOutage
+      annotations:
+          message: CRITICAL APPLICATION SLOWNESS. The 95th percentile (P95) of request durations is above 2 seconds for 5 consecutive minutes, indicating widespread user impact in {{ $labels.exported_namespace }}.
+          runbook_url: https://dsdmoj.atlassian.net/wiki/spaces/CDPT/pages/5124292758/Alerts+PHP+NodeJS+runbooks#SlownessOutage
+          summary: Critical application slowness (High P95 latency).
+      expr: |
+          histogram_quantile(
+            0.95,
+            sum by (le, ingress) (
+              rate(
+                nginx_ingress_controller_request_duration_seconds_bucket{exported_namespace="intranet-production"}[5m]
+              )
+            )
+          ) > 2
+      for: 5m
+      labels:
+          severity: care-arrangement-plan-production-alerts
+
+    - alert: High4xxRate
+      annotations:
+        message: More than 5% of responses were 4xx errors (excluding 404 Not Found) in the last 3 minutes. This may indicate broken links or authentication issues.
+        runbook_url: See README document in care-arrangement-plan GitHub repository for more details. 
+        summary: High rate of client errors (4xx).
+      expr: |
+        sum by (ingress, cluster) (rate(nginx_ingress_controller_requests{exported_namespace="care-arrangement-plan-prod",status=~"4[012356789].", status!="404"}[1m]))
+        /
+        sum by (ingress) (rate(nginx_ingress_controller_requests{exported_namespace="care-arrangement-plan-prod"}[1m]))
+        > 0.05
+      for: 3m
+      labels:
+        severity: care-arrangement-plan-production-alerts
+
+
+  - name: elasticache-rules
+    rules:
+    - alert: ElastiCacheCPUUtilizationHigh
+      expr: aws_elasticache_cpuutilization_average{cache_cluster_id=~"cp-f177eceeaf4eb770-00[12]"} > 70
+      for: 5m
+      labels:
+        severity: care-arrangement-plan-production-alerts
```

Comment:
I think, throughout his file, severity should be `care-arrangement-plan-alerts` - to match what was in the request https://github.com/ministryofjustice/cloud-platform/issues/7838 :)